source/backend/api/Areas/Reports/Controllers/AcquisitionController.cs

```diff
@@ -74,6 +83,44 @@ public IActionResult ExportLeases([FromBody] AcquisitionReportFilterModel filter
 
             return ReportHelper.GenerateExcel(reportAgreements, "Agreement Export");
         }
+
+        [HttpGet]
+        [HasPermission(Permissions.AcquisitionFileView)]
+        [Produces(ContentTypes.CONTENTTYPEEXCELX)]
+        [ProducesResponseType(200)]
+        [ProducesResponseType(204)]
+        [SwaggerOperation(Tags = new[] { "acquisitionfile", "report" })]
+        public IActionResult ExportAcquisitionFiles([FromQuery] AcquisitionFilterModel filter)
+        {
+            _logger.LogInformation(
+                "Request received by Controller: {Controller}, Action: {ControllerAction}, User: {User}, DateTime: {DateTime}",
+                nameof(AcquisitionController),
+                nameof(ExportAcquisitionFiles),
+                _user.GetUsername(),
+                DateTime.Now);
+
+            _logger.LogInformation("Dispatching to service: {Service}", _acquisitionFileService.GetType());
+
+            filter.ThrowBadRequestIfNull($"The request must include a filter.");
+            if (!filter.IsValid())
+            {
+                throw new BadRequestException("Acquisition files filter must contain valid values.");
+            }
+
+            var acceptHeader = (string)Request.Headers["Accept"];
+            if (acceptHeader != ContentTypes.CONTENTTYPEEXCEL && acceptHeader != ContentTypes.CONTENTTYPEEXCELX)
+            {
+                throw new BadRequestException($"Invalid HTTP request header 'Accept:{acceptHeader}'.");
+            }
+
+            var acquisitionFileData = _acquisitionFileService.GetAcquisitionFileExport((AcquisitionFilter)filter);
+            if (acquisitionFileData.Count.Equals(0))
```

Comment:
nit: not sure if we would need to check for acquisitionFileData to not be null?

Reply:
it doesn't seem so. It just returns an empty list instead of a null.